cli/command/service/inspect_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/docker/docker/api/types/container"
+
```

Comment:
nit: could you remove this blank line, and sort the imports in this group?

Reply:
It is done. I removed the blank line and sorted the imports.